notebooks/supernova_model_sims.ipynb

```diff
@@ -9,72 +9,298 @@
    "source": [
```

Comment:
<p>this looks like a very odd light curve. Do you know what's happening here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Not sure why it looks like that. You get the same lightcurve using the old implementation of the supernova class as well, so I guess it is not a problem, otherwise Justin would have seen it way earlier.